tests/apps/schedulerapp_grpc/.cache-include

```diff
@@ -0,0 +1 @@
+../utils/*.go
```

Comment:
nit: end file with newline

Reply:
none of the other `.cache-include` files have a newline from what I saw

Reply:
All text files should end in a newline :slightly_smiling_face: